llama_index/llms/ollama.py

```diff
@@ -57,6 +52,8 @@ def metadata(self) -> LLMMetadata:
             context_window=self.context_window,
             num_output=DEFAULT_NUM_OUTPUTS,
             model_name=self.model,
+            is_chat_model=True,  # Ollama supports chat API for all models
+            is_function_calling_model=True,  # Ollama supports json format outputs
```

Comment:
I think there's a difference between json outputs and function calling.

`is_function_calling_model` is intended to be a marker for models that use openais function calling format. Does ollama support a similar API with `tool_choices` ?

Reply:
It does not support tool_choices yet. We'll probably need to implement the json schema enforcer like litellm. For now let me set it to false.